api/client/gro_client.py

```diff
@@ -1082,9 +1111,16 @@ def GDH(self, gdh_selection, **optional_selections):
         entity_ids = [int(x) for x in gdh_selection.split("-")]
         selection = zip_selections(entity_ids, **optional_selections)
 
-        self.add_single_data_series(selection)
+        gdh_series = self.get_data_series(**selection)
+        if (len(gdh_series) > 1):
+            self._logger.warn("Multiple series returned for GDH selection")
+            return pandas.DataFrame()
+        for series in gdh_series: self.add_single_data_series(series)
+        entity_names = [series[entity_id.replace('id', 'name')] \
```

Comment:
There's a couple failing unit tests on this PR.

One of them's related to this line, breaking GDH:

```py
In [4]: client.GDH('860032-274-1215-0-2-9')                                                         
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
<ipython-input-4-6dcbd49e07c8> in <module>
----> 1 client.GDH('860032-274-1215-0-2-9')

~/api-client/api/client/gro_client.py in GDH(self, gdh_selection, **optional_selections)
   1118         for series in gdh_series: self.add_single_data_series(series)
   1119         entity_names = [series[entity_id.replace('id', 'name')] \
-> 1120                         for entity_id in DATA_SERIES_UNIQUE_TYPES_ID]
   1121 
   1122         try:

~/api-client/api/client/gro_client.py in <listcomp>(.0)
   1118         for series in gdh_series: self.add_single_data_series(series)
   1119         entity_names = [series[entity_id.replace('id', 'name')] \
-> 1120                         for entity_id in DATA_SERIES_UNIQUE_TYPES_ID]
   1121 
   1122         try:

NameError: free variable 'series' referenced before assignment in enclosing scope
```